.travis.yml

```diff
@@ -27,6 +27,7 @@ before_install:
   - rvm use 2.1.5 --install
   - nvm install 0.12.2
   - nvm use 0.12.2
+  - ./transform.sh
```

Comment:
If you just want to see the fully transformed code I can do that, but it is as of right now it ends up being

```
 593 files changed, 1481 insertions(+), 1481 deletions(-)
```